### PR TITLE
[Docs] Adding community guides page

### PR DESCRIPTION
**Pull request by dotNetkow**
- Add Community Guides page under "Guides" section to highlight good community content.

- Update Filesystem API page to include link to JM's post on how to use the Filesystem API.

- Update Geolocation API page to include link to BendyWorks Geolocation post.

### CODE REVIEW

File: site/docs-md/guides/community.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+## Integrations
+
+[Using Google Maps and Geolocation in Ionic with Capacitor](https://www.joshmorony.com/using-google-maps-and-geolocation-in-ionic-with-capacitor)
+
+[Create Native HTML5 Games with Phaser and Capacitor](https://www.joshmorony.com/create-native-html5-games-with-phaser-and-capacitor)
+
+[Integrating the Scanbot SDK in a Capacitor App with Ionic](https://scanbot.io/blog/capacitor-app/)
+
+[Sending Push Notifications with the PhoneGap Push Plugin in Capacitor](https://www.joshmorony.com/sending-push-notifications-with-the-phonegap-push-plugin-in-capacitor/)
```
Comment (jcesarmobile) [inline site/docs-md/guides/community.md]: This one probably doesn't work anymore, so I wouldn't link it

File: site/docs-md/apis/filesystem/index.md [hunk 2]
```diff
@@ -15,6 +15,8 @@ leave out the `directory` param to use a full file path.
 
 ## Example
 
+See also [Using the Capacitor Filesystem API to Store Photos](https://www.joshmorony.com/using-the-capacitor-filesystem-api-to-store-photos/).
```
Comment (jcesarmobile) [inline site/docs-md/apis/filesystem/index.md]: this one doesn't seem up to date neither

File: site/docs-md/guides/community.md (hunk 1 above)
Comment (dotNetkow) [reply site/docs-md/guides/community.md]: removed

File: site/docs-md/apis/filesystem/index.md (hunk 2 above)
Comment (dotNetkow) [reply site/docs-md/apis/filesystem/index.md]: removed

**Comment by dotNetkow** [comment]
Other links have been verified, yes.

**Pull request merged**